x-pack/plugins/security/server/authentication/index.test.ts

```diff
@@ -336,4 +337,57 @@ describe('setupAuthentication()', () => {
       );
     });
   });
+
+  describe('createApiKey()', () => {
```

Comment:
__note:__ I agree with @kobelb it probably would be enough to just `jest.mock('./api_key')` here and have just one test that checks that when we call `createAPIKey` returned from `setupAuthentication` we call underlying `createAPIKey` with proper arguments and return exactly what underlying method returns. No need to duplicate tests for the login itself.